onnxruntime/test/testdata/custom_op_library/custom_op_library.cc

```diff
@@ -45,13 +45,15 @@ struct KernelOne {
       out[i] = X[i] + Y[i];
     }
 #endif
+    return nullptr;
   }
 };
 
-// legacy custom op registration
-struct CustomOpOne : Ort::CustomOpBase<CustomOpOne, KernelOne> {
-  void* CreateKernel(const OrtApi& /* api */, const OrtKernelInfo* /* info */) const {
-    return std::make_unique<KernelOne>().release();
+// legacy custom op registration with kernel creation and compute function that return an OrtStatusPtr
+struct CustomOpOne : Ort::CustomOpBase<CustomOpOne, KernelOne, true> {
```

Comment:
@thiagocrepaldi @pranavsharma Please note that this test is already using the new API. The only part missing would be a test where the kernel creation or the compute method returns a non-null status.

Reply:
This should be fine.

Reply:
Ok, then I'll leave it at that!